<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Quicksti_Samday_FullRefunds_Admin" time="1.26" tests="1" failures="0" errors="1">
   <testsuite name="Quicksti_Samday_FullRefunds_Admin" tests="1" failures="0" errors="1" time="1.26" timestamp="2020-10-16 11:38:17" hostname="admin - SYS8" id="Test Suites/Certpay_testSuite/Quicksti_Samday_FullRefunds_Admin">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="E:\\Sush_Certpay\\Reports\\20201016_113813\\Certpay_testSuite\\Quicksti_Samday_FullRefunds_Admin\\20201016_113813"/>
         <property name="logFiles" value="E:\\Sush_Certpay\\Reports\\20201016_113813\\Certpay_testSuite\\Quicksti_Samday_FullRefunds_Admin\\20201016_113813\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="admin - SYS8"/>
         <property name="hostAddress" value="172.31.253.251"/>
         <property name="os" value="Windows 10 32bit"/>
         <property name="katalonVersion" value="7.1.2.1"/>
      </properties>
      <testcase name="Test Cases/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund" time="0.832" classname="Test Cases/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund" status="ERROR">
         <error type="ERROR" message="Test Cases/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: The current scope already contains a variable of the name RefundAmt
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: The current scope already contains a variable of the name ConvFee
 @ line 161, column 10.
   	Println ConvFee
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: unable to resolve class Println 
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: unable to resolve class Println 
 @ line 161, column 10.
   	Println ConvFee
            ^

4 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1602828493612.run(TempTestSuite1602828493612.groovy:36)
"/>
         <system-out><![CDATA[2020-10-16 11:38:17 - [TEST_CASE][ERROR] - Test Cases/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund: Test Cases/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: The current scope already contains a variable of the name RefundAmt
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: The current scope already contains a variable of the name ConvFee
 @ line 161, column 10.
   	Println ConvFee
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: unable to resolve class Println 
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: unable to resolve class Println 
 @ line 161, column 10.
   	Println ConvFee
            ^

4 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1602828493612.run(TempTestSuite1602828493612.groovy:36)


2020-10-16 11:38:18 - [TEST_STEP][PASSED] - Start listener action : beforeTestCase: null

2020-10-16 11:38:18 - [TEST_STEP][PASSED] - Start listener action : afterTestCase: Browser is closed

2020-10-16 11:38:18 - [TEST_STEP][PASSED] - closeBrowser(): Browser is closed

2020-10-16 11:38:18 - [MESSAGE][PASSED] - Browser is closed]]></system-out>
         <system-err><![CDATA[2020-10-16 11:38:17 - [TEST_CASE][ERROR] - Test Cases/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund: Test Cases/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: The current scope already contains a variable of the name RefundAmt
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: The current scope already contains a variable of the name ConvFee
 @ line 161, column 10.
   	Println ConvFee
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: unable to resolve class Println 
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: unable to resolve class Println 
 @ line 161, column 10.
   	Println ConvFee
            ^

4 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1602828493612.run(TempTestSuite1602828493612.groovy:36)]]></system-err>
      </testcase>
      <system-out><![CDATA[2020-10-16 11:38:17 - [TEST_SUITE][ERROR] - Quicksti_Samday_FullRefunds_Admin: Test Cases/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: The current scope already contains a variable of the name RefundAmt
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: The current scope already contains a variable of the name ConvFee
 @ line 161, column 10.
   	Println ConvFee
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: unable to resolve class Println 
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: unable to resolve class Println 
 @ line 161, column 10.
   	Println ConvFee
            ^

4 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1602828493612.run(TempTestSuite1602828493612.groovy:36)]]></system-out>
      <system-err><![CDATA[2020-10-16 11:38:17 - [TEST_SUITE][ERROR] - Quicksti_Samday_FullRefunds_Admin: Test Cases/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: The current scope already contains a variable of the name RefundAmt
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: The current scope already contains a variable of the name ConvFee
 @ line 161, column 10.
   	Println ConvFee
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 159: unable to resolve class Println 
 @ line 159, column 10.
   	Println RefundAmt
            ^

file:/E:/Sush_Certpay/Scripts/SameDay_Refunds_Admin/TC_001_VISA_Sameday_FullRefund/Script1599466043016.groovy: 161: unable to resolve class Println 
 @ line 161, column 10.
   	Println ConvFee
            ^

4 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1602828493612.run(TempTestSuite1602828493612.groovy:36)]]></system-err>
   </testsuite>
</testsuites>
